application/microservices/entando-hub-catalog-ms/src/main/java/com/entando/hub/catalog/rest/BundleGroupVersionController.java

```diff
@@ -157,7 +157,7 @@ public ResponseEntity<BundleGroupVersion> updateBundleGroupVersion(@PathVariable
     @GetMapping(value = "/versions/{bundleGroupId}",produces = {"application/json"})
     @ApiResponse(responseCode = "400", description = "Bad Request", content = @Content)
     @ApiResponse(responseCode = "200", description = "OK")
-    public PagedContent<BundleGroupVersionFilteredResponseView, com.entando.hub.catalog.persistence.entity.BundleGroupVersion> getBundleGroupVersions(@PathVariable String bundleGroupId, @RequestParam Integer page, @RequestParam Integer pageSize, @RequestParam(required = false) String[] statuses) {
+    public PagedContent<BundleGroupVersionFilteredResponseView, com.entando.hub.catalog.persistence.entity.BundleGroupVersion> getBundleGroupVersions(@PathVariable Long bundleGroupId, @RequestParam Integer page, @RequestParam Integer pageSize, @RequestParam(required = false) String[] statuses) {
```

Comment:
Please add an import for the BundleGroupVersion class to simplify this method definition.

Reply:
As agreed during the meeting we decided to address these changes in a dedicated PR in the next days.